frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -186,7 +186,6 @@ export function DashboardItem({
     const [initialLoaded, setInitialLoaded] = useState(false)
     const [showSaveModal, setShowSaveModal] = useState(false)
     const { dashboards } = useValues(dashboardsModel)
-    /* FIXME (see #5454): const { refreshStatus } = useValues(dashboardLogic) */
```

Comment:
We're not mounting dashboardLogic here anymore, which will get rid of those `prop.key is undefined` errors in home.